src/components/wallet/methods/signOut.ts

```diff
@@ -1,22 +1,19 @@
 import { remove } from '@services/storage'
 import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
-import { PromptInput } from '../../prompt/promptInput'
+import { PromptInput } from '@prompt/promptInput'
 
 export default async function signOut(this: Wallet) {
   try {
-    const inputs = await this.setPrompt({
-      message: 'Are you sure? This will nuke your account',
-      inputs: [new PromptInput('Enter NUKE to confirm')],
+    await this.setPrompt({
+      message: 'You can press the back button to sign back in.',
```

Comment:
Minor messaging suggestion.

```suggestion
      message: 'You can enter your Secret Key or press back in your browser to sign back in.',
```

Reply:
```suggestion
      message: 'You can add your Secret Key to the URL or press back in your browser to sign back in.',
```

Reply:
Looks good to me!